parser.js

```diff
@@ -1365,7 +1365,7 @@
         // NestedClass ::
         //      ...
         //      \ CharacterClassEscape[+U, +V]
-        if (res = parseCharacterClassEscape()) {
+        if (res = parseClassEscape()) {
```

Comment:
Just checking, was that a bug to parse as `parseCharacterClassEscape` before?

Reply:
Yes, because it didn't accept `\u` escapes. `parseCharacterClassEscape` is used to parse escapes that _result_ in a character class, such as `\d` which is `[0-9]`.

Reply:
Okay, thanks for the details!